ci/docker-read-local-label.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 IMAGE LABEL"
+    echo "Example: $0 elifesciences/annotations_cli org.elifesciences.dependencies.api-dummy"
+    exit 1
+fi
+
+image="${1}"
+label="${2}"
+docker inspect "${image}" | jq -r ".[0].Config.Labels[\"${label}\"]"
```

Comment:
you can probably inline `docker inspect ... | jq ...` wherever you need it, if this doesn't use the pulling part

Reply:
Still true: this is essentially a single line so can be put in place of https://github.com/elifesciences/sciencebeam-trainer-grobid/pull/2/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R33 directly as a `bash -c ""`? If quoting/backslashing is not too complicated.

Reply:
It's just more difficult to test outside Jenkins. And especially the additional Groovy string encoding can make it make it fall over.